docs/integrate/rsyslog/tutorial.md

```diff
@@ -0,0 +1,140 @@
+(rsyslog-tutorial)=
+# Store server logs on CrateDB for fast search and aggregations
+
+## Introduction
+
+CrateDB stores server logs efficiently and makes them easy to query.
+
+Common pain points with traditional log stacks and SIEMs include:
+
+* timeouts when searching across long time ranges
+* proprietary, complex query syntaxes
+* awkward integrations with application monitoring dashboards
+
+CrateDB addresses these issues: query logs with standard SQL from any
+PostgreSQL‑compatible tool, and use full‑text search and aggregations
+backed by efficient indexes. The sections below walk through a minimal
+setup.
+
+## Setup
+
+### CrateDB
+
+First, start CrateDB. For production, use a dedicated cluster. For this demo, run a single‑node container:
+
+```bash
+sudo docker run -d --name cratedb \
+  -p 4200:4200 -p 5432:5432 \
+  -e CRATE_HEAP_SIZE=1g \
+  crate:latest -Cdiscovery.type=single-node
+```
```

Comment:
@kneth: What do you think about omitting the `sudo` all around, so the tutorial commands can easily used more universally, e.g. on macOS, without much ado?